src/modules/dashboard/components/ColonyFundingMenu/ColonyFundingMenu.tsx

```diff
@@ -0,0 +1,125 @@
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
+import React, { useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+
+import {
+  ColonyTokenEditDialog,
+  TokenMintDialog,
+  TokensMoveDialog,
+} from '~admin/Tokens';
+import Button from '~core/Button';
+import { useDialog } from '~core/Dialog';
+import { Colony, useLoggedInUser } from '~data/index';
+import { useTransformer } from '~utils/hooks';
+
+import { getUserRolesForDomain } from '../../../transformers';
+import { userHasRole } from '../../../users/checks';
```

Comment:
i feel we need to create aliases for this

Reply:
For the user checks you mean ?

Reply:
and transformers, to get rid of "../../../"

Reply:
Sure, I'll think of a way.

Just that we're going to have a lot of them by the end...

That list is already pretty long:
https://github.com/JoinColony/colonyDapp/blob/feature/dev-17/webpack.base.js#L40-L50

Plus the ones that are auto-generated:
https://github.com/JoinColony/colonyDapp/blob/feature/dev-17/webpack.base.js#L20-L31

But I think I can hook into the generator and make it do what we want auto-magically :sparkles: 

:slightly_smiling_face: